modules/mw.cx.init.Translation.js

```
@@ -393,6 +393,12 @@ mw.cx.init.Translation.prototype.fetchDraftTranslation = function (
  * @return {Promise<mw.cx.dm.DraftTranslationDTO|null>} Draft or null.
  */
 mw.cx.init.Translation.prototype.fetchDraftTranslationSuccess = function (draft, conflict) {
+	const admin_users = ['Doc James', 'Mr. Ibrahem 1'];
```

ESLint

[error] Identifier 'admin_users' is not in camel case.

(camelcase)